.gitignore

```diff
@@ -77,3 +77,4 @@ lint/generated/
 lint/outputs/
 lint/tmp/
 # lint/reports/
+/.idea/
```

Comment:
`.idea/` is already ignored in this `.gitignore` you have to remove the files that you are already tracking from the index. See [this SO post for more details](https://stackoverflow.com/questions/25436312/gitignore-not-working).

Reply:
And the config files aren't ignored yet.